tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -572,6 +572,77 @@ def test_callf_stack_overflow_by_height(eof_test, callee_stack_height):
     eof_test(container=container, expect_exception=EOFException.STACK_OVERFLOW)
 
 
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="underflow_1",
+            sections=[
+                Section.Code(
+                    code=Op.CALLF[1] + Op.STOP,
+                    max_stack_height=1,
+                ),
+                Section.Code(
+                    code=Op.PUSH0 + Op.RETF,
+                    code_inputs=1,
+                    code_outputs=2,
+                    max_stack_height=2,
+                ),
+            ],
+            expected_bytecode="ef000101000802000200040002040000000080000101020002e30001005fe4",
+        ),
+        Container(
+            name="underflow_variable_stack_2",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[2]
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.CALLF[1]
+                    + Op.STOP,
+                    max_stack_height=4,
+                ),
+                Section.Code(
+                    code=Op.PUSH0 + Op.RETF,
+                    code_inputs=4,
+                    code_outputs=5,
+                    max_stack_height=5,
+                ),
+            ],
+            expected_bytecode="ef0001010008020002000c00020400000000800004040500055f6000e100025f5fe30001005fe4",
+        ),
+        Container(
+            name="underflow_variable_stack_3",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
+                    + Op.RJUMPI[2]
+                    + Op.PUSH0
+                    + Op.PUSH0
+                    + Op.CALLF[1]
+                    + Op.STOP,
```

Comment:
Might be worth it to add another test case where the underflow happens in the no-jump branch:
```python
code=Op.PUSH0
                    + Op.PUSH1[0]
                    + Op.PUSH1[0]
                    + Op.PUSH1[0]
                    + Op.RJUMPI[1]
                    + Op.POP
                    + Op.CALLF[1]
                    + Op.STOP,
```

Reply:
Added as "underflow_variable_stack_4".